pythonosc/dispatcher.py

```diff
@@ -44,17 +42,17 @@ def handlers_for_address(self, address_pattern):
     # we're fine.
     pattern = pattern + '$'
     pattern = re.compile(pattern)
-    matched = [
-        handler for addr, handler in self._map.items()
-        if ( pattern.match(addr) # wildcards could be in the incoming pattern
-             # .. or the mapped pattern
-             or (('*' in addr) and re.match(addr.replace('*','[^/]*?/*'), address_pattern))
-           )
-        ]
+    matched = False
+
+    for addr, handlers in self._map.items():
+      if pattern.match(addr) \
```

Comment:
just a style concern but I prefer avoiding the use of \, prefer (instead.

if (pattern.match(addr)
    or (('*' in addr) and re.match(addr.replace('*','[^/]*?/*'), address_pattern)):